crates/router/src/core/admin.rs

```diff
@@ -874,3 +871,210 @@ pub fn get_frm_config_as_secret(
+            Ok(())
+        }
+        api_enums::Connector::Iatapay => {
+            iatapay::transformers::IatapayAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Klarna => {
+            klarna::transformers::KlarnaAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Mollie => {
+            mollie::transformers::MollieAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Multisafepay => {
+            multisafepay::transformers::MultisafepayAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Nexinets => {
+            nexinets::transformers::NexinetsAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Nmi => {
+            nmi::transformers::NmiAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Noon => {
+            noon::transformers::NoonAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Nuvei => {
+            nuvei::transformers::NuveiAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Opennode => {
+            opennode::transformers::OpennodeAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Payme => {
+            payme::transformers::PaymeAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Paypal => {
+            paypal::transformers::PaypalAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Payu => {
+            payu::transformers::PayuAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Powertranz => {
+            powertranz::transformers::PowertranzAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Rapyd => {
+            rapyd::transformers::RapydAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Shift4 => {
+            shift4::transformers::Shift4AuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Stax => {
+            stax::transformers::StaxAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Stripe => {
+            stripe::transformers::StripeAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Trustpay => {
+            trustpay::transformers::TrustpayAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Tsys => {
+            tsys::transformers::TsysAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Wise => {
+            wise::transformers::WiseAuthType::try_from(val)?;
+            Ok(())
+        }
+        api_enums::Connector::Worldline => {
+            worldline::transformers::AuthType::try_from(val)?;
```

Comment:
@ArjunKarthik should we consider renaming this as `WorldlineAuthType`?

Reply:
Yeah @SanchithHegde